### DISCUSSION TITLE
How do I change the text highlight/select color?

### DISCUSSION THREAD

**Discussion by ElectricRCAircraftGuy** [mmistakes/minimal-mistakes#4392] [Q&A] [answered]
When you click and drag with your mouse, it selects text. How do you set the select color that the text gets highlighted with?

**Comment by ElectricRCAircraftGuy** [comment]
See here too: https://stackoverflow.com/q/76499711/4561887

**Comment by ENTPRESTIGIOUS** [comment]
Just create a custom CSS and use this as a reference: https://codepen.io/soberdash/pen/ogoLMO

Oh it's been already answered! Nice!

**Comment by ElectricRCAircraftGuy** [comment, marked as answer]
Yeah, we got it now!

See my question here: [CSS: How do you inspect the settings for text you are highlighting/selecting with your mouse or keyboard?](https://stackoverflow.com/q/76499711/4561887). It now has 3 helpful answers, including [a thorough answer I wrote](https://stackoverflow.com/a/76502482/4561887).
